.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -3,13 +3,13 @@
 
 ## Checklist
 
- - [ ] Added an entry in `CHANGES.txt` for user facing changes
+ - [ ] Added an entry in the latest `<upcoming_version>.rst` for user facing changes
```

Comment:
maybe `X.X.0.rst` is better?

Reply:
yep, I'd go with `<x.y.0>` and maybe mention the release-notes folder?

Reply:
thx! please check again.